cmd/hegel/root_cmd.go

```diff
@@ -36,9 +49,10 @@ Deprecated CLI flags
 Deprecated environment variables
   Deprecated          Current
   DATA_MODEL_VERSION  HEGEL_DATA_MODEL
+  GRPC_PORT           HEGEL_GRPC_PORT
+  HEGEL_USE_TLS       HEGEL_GRPC_USE_TLS
```

Comment:
`HEGEL_USE_TLS` should go after `HEGEL_TLS_KEY` to maintain sorted order.

Reply:
This isn't sorted, hence custom endpoints is at further down. The reason for this change was for humans given we read top to bottom in English and you likely want to know if TLS is enabled before worrying about the configuration of TLS. That theme is in the options so appears here to. 

I'm good with either approach, let me know.

Reply:
I ordered it. Resolved.